test/3D_laplacian.jl

```diff
@@ -1,21 +1,36 @@
-using DiffEqOperators
-
+using DiffEqOperators, DifferentialEquations, ProgressMeter
+function main()
 s = x, y, z = (-5:0.2:5, -5:0.2:5, -5:0.2:5)
 dx = dy = dz = x[2] - x[1]
 
+ricker(x::T, y::T, z::T) where T = (4*(x^2+y^2+z^2) - 6)*exp(-(x^2+y^2+z^2))
+
+u0 = [ricker(X, Y, Z) for Z in z, Y in y, X in x]
+
 Dxx = CenteredDifference{1}(2, 4, dx, length(x))
 Dyy = CenteredDifference{2}(2, 4, dy, length(y))
 Dzz = CenteredDifference{3}(2, 4, dz, length(z))
-Q = compose(PeriodicBC(Float64,length.(s))...)
 
-∇² = Dxx + Dyy + Dzz
+A = Dxx+Dyy+Dzz
+Q = compose(Dirichlet0BC(Float64, length.(s))...)
 
-gaussian(x::T, y::T, z::T) where T = exp(-(x^2 + y^2+ z^2))
-ricker(x::T, y::T, z::T) where T = (4*(x^2+y^2+z^2) - 6)*exp(-(x^2+y^2+z^2))
-u0 = [gaussian(X, Y, Z) for Z in z, Y in y, X in x]
-u0xx = [ricker(X, Y, Z) for Z in z, Y in y, X in x]
-A = (∇² * Q)
-u_prime = A * u0
+dt = dx/(sqrt(3)*3e8)
+t = 0.0:dt:10/3e8
+
+f(u,p,t) = (3e8)^2 .*(A*Q*u)
+
+uolder = deepcopy(u0)
+uold = deepcopy(u0)
+u = deepcopy(u0)
+
+function steptime(u,uold,uolder)
+    return ((dt^2 .*f(u,0,0) .+ 5u .- 4uold .+ uolder)./2, u, uold)
```

Comment:
we need to add this method :(

Reply:
Do we still need this method? If so I'll try write something general like the RobinBC constructor. 

Isn't this just an Integrator though?


-

Reply:
The issue is that we're missing this IMEX method in OrdinaryDiffEq. It would be good to add.